examples/basic-recipe.rs

```diff
@@ -26,21 +26,17 @@ fn load_recipe() -> Result<Backend, String> {
 
     let recipe = soup.migrate(|mig| {
         // install recipe
-        match Recipe::from_str(&sql, None) {
-            Ok(mut recipe) => {
-                recipe.activate(mig, false)?;
-                recipe
-            }
-            Err(e) => return Err(e),
-        }
+        let mut recipe = Recipe::from_str(&sql, None).unwrap();
+        recipe.activate(mig, false).unwrap();
```

Comment:
I changed this to using `.unwrap()` as I think it makes the example more readable. Let me know if you disagree and I'll change it back to returning a `Result`